app/_src/mesh/guides/jwt-validation.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Use OPA to validate JWT tokens
+content-type: tutorial
+description: Validate JWT tokens using an OPA policy in Kong Mesh.
+---
+
+A companies commonly face the following security problems:
+* How to determine that a user is who they say they are when logging in
+* How to ensure that only authorized users have access to certain applications and enforce security policies over API requests and for upstream services. 
+
+These problems can be solved by using a {{site.mesh_product_name}} [MeshOPA (beta)](/mesh/{{page.kong_version}}/features/meshopa/) policy to authorize JWT tokens by using [Envoy's External authorization filter](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ext_authz_filter.html) with [OPA](https://www.openpolicyagent.org/docs/latest/envoy-introduction/). Keep in mind that OPA doesn't handle authentication, it only handles authorization.
+
+This guide explains how to configure and deploy a MeshOPA policy that authorizes users based on if they have a valid JWT token. <!--what are the advantages of this over other options?-->.
```

Comment:
@lahabana One question I had is why would a customer want to validate JWT this way instead of another way? I know OPA is easier than another policy, but why specifically with a Mesh policy?